docs/extensions/flex.md

```diff
@@ -0,0 +1,66 @@
+
+<img src="../assets/flex-userjourney-resize.jpg" alt="GTFS-Flex User Journey">
+
+🔮 MobilityData expects GTFS-Flex to open the door to deeper standardization of demand responsive transportation including expansion into transactional and real-time components using GTFS-OnDemand. We are preparing a suggested strategy to best handle the growing number of modes of transportation and complexity of concepts engaging in this area.
+
+<a class="button no-icon" href="https://github.com/google/transit/issues/382" target="_blank">See the full proposal</a>
+
+## WIP: Base Implementation
+We have opened an issue to cover the base implementation of GTFS-Flex. Using this incremental approach should allow us to better establish a strong base for consensus.
+
+🔭 Scope of GTFS-Flex: Base Implementation
+
+Based on what has already been tested in the field, we see GTFS-Flex solving the following key use cases:
+
+- Display available demand responsive transportation services for rider discovery
+- Deviated bus routes
+- Dial-a-ride (e.g Rufbus, On-call bus)
+- Displaying contact information (phone number and/or website URL) for booking
+
+<a class="button no-icon" href=https://share.mobilitydata.org/slack>Join #gtfs-flex on Slack</a><a class="button no-icon" href=https://groups.google.com/g/gtfs-changes>Join GTFS-Changes Google Group</a>
+
+## Early Implementations
+
+- [MNDoT Flex Pilot Project: Trillium, IBI, Transit, MNDoT, Cambridge Systematics, and Token Transit](https://blog.transitapp.com/case-study/mndot-gtfs-flex-bringing-rural-riders-into-the-fold/) 
+- [Open Trip Planner](https://www.opentripplanner.org/)
+- [Mobility on Demand Sandbox in Vermont](https://www.connectingcommuters.org/)
+- [Tulare County Area Transit](https://ridetcat.org/)
+- [Northwest Oregon Transit Alliance (NW Connector)](https://www.nworegontransit.org/)
+- [Vamos Mobility App](https://vamosmobileapp.com/)
+- [RTD Denver Flexride](https://www.rtd-denver.com/services/flexride)
+
+Contact us to add your GTFS-Flex implementation to this page
+
+<a class="button no-icon" href=https://share.mobilitydata.org/slack>Contact us</a>
+
+## Adoption tracker
+### Current
+
+<iframe class="airtable-embed" src="https://airtable.com/embed/shrUPyCZWOWrvO2mX?backgroundColor=purple&viewControls=on" frameborder="0" onmousewheel="" width="100%" height="533" style="background: transparent; border: 1px solid #ccc;"></iframe>
+
+<a class="button no-icon" href="https://airtable.com/shrcac1fXUrMxfoDV" target="_blank">Request a change</a><a class="button no-icon" href="https://airtable.com/shrgnVR5Su9tkHvUv" target="_blank">Add your organization (consumers)</a><a class="button no-icon" href="https://airtable.com/shrsU4idBtcLuRuwZ" target="_blank">Add your organization (producers)</a>
+
+### Future
```

Comment:
suggestion: Looks like there's no current Future info - I'd suggest removing it from the page until there's actual submissions.